src/Docs/Resources/current/4-how-to/670-administration-language-switcher.md

```diff
@@ -0,0 +1,256 @@
+[titleEn]: <>(Switching languages in administration and storefront)
+[metaDescriptionEn]: <>(This HowTo will teach you how the editing language is changed and what impacts these changes have.)
+
+## Overview
+
+The editing language of the administration is one of the key USPs of its user interface.
+It is present all over the application.
+The most important fact is that it is used change for everything at once to keep editing consistent.
+For example if you change the editing language in the language dropdown every listing or detail page will update and use that new language to display and edit content.
+To make sure your custom entity supports translations look at the [overview of translations in the DAL](../../2-internals/1-core/20-data-abstraction-layer/120-translations.md). 
+
+
+## Change language in a listing
+
+To change the language in an administration listing you have to add the language switching component into your action bar:
+
+```twig
+{# foobar-list.html.twig #}
+
+<sw-page>
+    <template slot="language-switch">
```

Comment:
Please use `<template #language-switch>` for that. It will be required in the future :)